html-forms/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="email" id="email" name="email" />
+      <div>
+        <label for="why-is-coding-cool">Why is coding cool?</label>
+      </div>
+      <div>
+        <select id="why-is-coding-cool" name="why-is-coding-cool">
+          <option>Select an option</option>
+          <option>Because it's interesting</option>
+          <option>Because it's fun</option>
+          <option>Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <p>Favorite Language (choose one)</p>
+        <input type="radio" id="html" name="favorite-language" value="HTML" />
+        <label for="html">HTML</label>
+        <input type="radio" id="css" name="favorite-language" value="CSS" />
+        <label for="css">CSS</label>
+        <input
+          type="radio"
+          id="javascript"
+          name="favorite-language"
+          value="JavaScript" />
+        <label for="javascript">JavaScript</label>
+      </div>
+      <div>
+        <p>Would you like to sign up for more information?</p>
+        <input type="checkbox" id="sign-up" name="sign-up" value="yes" />
+        <label for="sign-up">Sign me up!</label>
+      </div>
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <div>
+        <textarea cols="30" rows="8"></textarea>
```

Comment:
This should have an `id="comments"` so it's associated with the `label` on line 46